common_docs_zh/ecosystem/fluentbit.md

```diff
@@ -0,0 +1,369 @@
+        at org.apache.doris.planner.OlapScanNode.addScanRangeLocations(OlapScanNode.java:931) ~[doris-fe.jar:1.2-SNAPSHOT]
+        at org.apache.doris.planner.OlapScanNode.computeTabletInfo(OlapScanNode.java:1197) ~[doris-fe.jar:1.2-SNAPSHOT]
+```
+
+**2. 建表**
+
+表结构包括日志的产生时间，采集时间，主机名，日志文件路径，日志类型，日志级别，线程名，代码位置，日志内容等字段。
+
+```
+CREATE TABLE `doris_log` (
+  `log_time` datetime NULL COMMENT 'log content time',
+  `collect_time` datetime NULL COMMENT 'log agent collect time',
+  `host` text NULL COMMENT 'hostname or ip',
+  `path` text NULL COMMENT 'log file path',
+  `type` text NULL COMMENT 'log type',
+  `level` text NULL COMMENT 'log level',
+  `thread` text NULL COMMENT 'log thread',
+  `position` text NULL COMMENT 'log code position',
+  `message` text NULL COMMENT 'log message',
+  INDEX idx_host (`host`) USING INVERTED COMMENT '',
+  INDEX idx_path (`path`) USING INVERTED COMMENT '',
+  INDEX idx_type (`type`) USING INVERTED COMMENT '',
+  INDEX idx_level (`level`) USING INVERTED COMMENT '',
+  INDEX idx_thread (`thread`) USING INVERTED COMMENT '',
+  INDEX idx_position (`position`) USING INVERTED COMMENT '',
+  INDEX idx_message (`message`) USING INVERTED PROPERTIES("parser" = "unicode", "support_phrase" = "true") COMMENT ''
+) ENGINE=OLAP
+DUPLICATE KEY(`log_time`)
+COMMENT 'OLAP'
+PARTITION BY RANGE(`log_time`) ()
+DISTRIBUTED BY RANDOM BUCKETS 10
+PROPERTIES (
+"replication_num" = "1",
+"dynamic_partition.enable" = "true",
+"dynamic_partition.time_unit" = "DAY",
+"dynamic_partition.start" = "-7",
+"dynamic_partition.end" = "1",
+"dynamic_partition.prefix" = "p",
+"dynamic_partition.buckets" = "10",
+"dynamic_partition.create_history_partition" = "true",
+"compaction_policy" = "time_series"
+);
+```
+
+**3. 配置**
+
+Fluent Bit 日志采集的配置文件如下，doris_log.conf 用于定义 ETL 的各个部分组件，parsers.conf 用于定义不同的日志解析器。
+
+doris_log.conf:
+```
+# config for Fluent Bit service
+[SERVICE]
+    log_level info
+    # parsers file
+    parsers_file parsers.conf
+
+# use input tail
+[INPUT]
+    name tail
+    path /path/to/your/log
+    # add log file name to the record, key is 'path'
+    path_key path
+    # set multiline parser
+    multiline.parser multiline_java 
+
+# parse log
+[FILTER]
+    match *
+    name parser
+    key_name log
+    parser fe_log
+    reserve_data true
+
+# add host info
+[FILTER]
+    name sysinfo
+    match *
+    # add hostname to the record, key is 'host'
+    hostname_key host
+
+# output to doris
+[OUTPUT]
+    name doris
+    match *
+    host fehost
+    port feport
+    user your_username
+    password your_password
+    database your_db
+    table your_table
+    # add 'collect_time' to the record
+    time_key collect_time
+    # 'collect_time' is timestamp, change it to datatime
+    header columns collect_time=from_unixtime(collect_time)
+    log_request true
+    log_progress_interval 10
+```
+
+parsers.conf
+```
```

Comment:
please add all the code fence for code block

Reply:
I checked to make sure that code fences are added for all code blocks.